crates/sui-types/src/messages_checkpoint.rs

```diff
@@ -125,9 +125,11 @@ pub struct CheckpointSummary {
     /// The running total gas costs of all transactions included in the current epoch so far
     /// until this checkpoint.
     pub epoch_rolling_gas_cost_summary: GasCostSummary,
-    /// If this checkpoint is the last checkpoint of the epoch, we also include the committee
-    /// of the next epoch. This allows anyone receiving this checkpoint know that the epoch
-    /// will change after this checkpoint, as well as what the new committee is.
+    /// next_epoch_committee has a non-null value if and only if the current checkpoint is
+    /// the last checkpoint of an epoch.
+    /// Therefore next_epoch_committee can be used to pick the last checkpoint of an epoch,
+    /// which is often useful to get epoch level summary stats like total gas cost of an epoch,
+    /// or the total number of transactions from genesis to the end of an epoch.
     /// The committee is stored as a vector of validator pub key and stake pairs. The vector
     /// should be sorted based on the Committee data structure.
     /// TODO: If desired, we could also commit to the previous last checkpoint cert so that
```

Comment:
Perhaps we can also make the TODO a non-doc comment while we're here--TODOs in code are fine, but not great to have them in public-facing docs.